simplq/src/services/queue.js

```diff
@@ -29,3 +30,10 @@ export const getStatusByName = async (queueName) => {
   const response = await makeRequest('get', `/queue/status?queueName=${queueName}`);
   return response.data;
 };
+
+export const deleteQueue = async (queueId) => {
+  const response = await makeRequest('delete', `/queue/${queueId}`);
+  await getMyQueues().then((myQueues) => store.dispatch(setMyQueues(myQueues)));
+  history.push('/');
+  return response.data;
```

Comment:
Did you miss adding the catch or should I add exception handling while adding the axios interceptor?

Reply:
Let's add in one place... if you are planning to do that soon